.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 /docs/ @KittyCAD/kcl
 /src/ @KittyCAD/frontend
 /rust/ @KittyCAD/kcl
+e2e/playwright/mecheng-approved-paths @nicboone8 @max-mrgrsk
```

Comment:
Thanks for adding this! Instead of `mechend-approved-paths/` how about something like `benchmark/`? Getting a dedicated GitHub group would be great as well but not a blocker.